src/Take.Blip.Builder/AnalyzeBuilderExceptions.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using Jint.Runtime;
+
+namespace Take.Blip.Builder
+{
+    /// <summary>
+    /// Defines a builder exception analyze service.
+    /// </summary>
+    public class AnalyzeBuilderExceptions : IAnalyzeBuilderExceptions
+    {
+        private List<Func<Exception, bool>> _validations;
+
+        /// <inheritdoc />
+        public AnalyzeBuilderExceptions()
+        {
+            _validations = new List<Func<Exception, bool>>()
+            {
+                ex => ex is ActionProcessingException && ex.InnerException != null && ex.InnerException is ArgumentException,
```

Comment:
Here, you can use `ex?.InnerException is ArgumentException` instead of `ex.InnerException != null && ex.InnerException is ArgumentException`.

Reply:
Nicee, Done!